### ISSUE TITLE
Accessible names in tables

### ISSUE THREAD

**Issue by obr42** [beer-garden/react-ui#353]
- Accessible name for sorting buttons are always `Toggle SortBy` - might be better to have column's name in it?
  - Rows per page dropdown has accessible name: `Rows per page: rows per page`
  - Toggle SortBy should have a description that says ascending/descending as applicable
  - Filter chips have no label
